vectors_statistics.md

```diff
@@ -0,0 +1,70 @@
+# Vectors Statistics
+
+## Default
+
+'highConfidenceWeightedVector' is being used:  
+
+Weighted feature vector generated from ModelTrainer.cs/ModelTrainer.exe during the training phase based on in-the-wild samples.  
+
+- Accuracy: 0.9605  
+- Precision: 0.9609  
+- Recall: 0.9353  
+- F1Score: 0.9479  
+- TruePositiveRate: 0.3595  
+- FalsePositiveRate: 0.0146  
+- TrueNegativeRate: 0.6010  
+- FalseNegativeRate: 0.0249  
+
+## -Deep
+
+*Specifies that the deeper (but lower confidence) weighted vector be used to measure input vector, thus a "deep" inspection that will allow more False Positives but fewer False Negatives than the default high confidence weighted vector.*
+
+'broadNetWeightedVector' is being used:  
+
+Weighted feature vector generated from ModelTrainer.cs/ModelTrainer.exe during the training phase.
+
+- Accuracy: 0.8926
+- Precision: 0.8977
+- Recall: 0.8849
+- F1Score: 0.8913
+- TruePositiveRate: 0.4403
+- FalsePositiveRate: 0.0502
+- TrueNegativeRate: 0.4523
+- FalseNegativeRate: 0.0573
+
+## -CommandLine
+
+*Specifies that the command-specific (as opposed to the default script-specific) weighted vector be used to measure input vector.*
+
+'commandLineWeightedVector' is being used:  
+
+Weighted feature vector generated from ModelTrainer.cs/ModelTrainer.exe during the training phase based on in-the-wild samples.  
+  
+- Accuracy: 0.9959  
```

Comment:
Do we believe these numbers?

Reply:
I'll let @danielbohannon decide if he wants to keep it :)